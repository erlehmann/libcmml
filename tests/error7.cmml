<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<!DOCTYPE cmml SYSTEM "cmml.dtd">

<cmml>

<head>
<title>
Australia Advances - Grim News on the Ozone Layer
</title>
<meta name="DESCRIPTION" content="The recovery of the ozone layer is being slowed dramatically by the presence and production of the gas, halon"/>
<meta name="KEYWORDS" content="Ozone, Halon"/>
<meta name="TYPE" content="video"/>
</head>

<clip id="intro" start="npt:0">
<a href="http://www.csiro.au/promos/ozadvances/">
http://www.csiro.au/promos/ozadvances/</a>
<desc>
Introduction to 'Grim News on the Ozone Layer - Australia Advances video'.
</desc>
</clip>

<stream basetime="npt:0">
<import contenttype="video/mpeg" src="GrimNewsOnOzoneB.mpg" start="npt:0"/>
</stream>

<clip id="startspeech" start="npt:9">
<desc>
 Just when you thought the Earth's ozone layer was safe
      from CFC chemicals, there's a new threat. Halon 1202. And while
      other damaging chemicals have been identified and banned, Halon
      1202 is not only increasing rapidly, we don't even know where
      it's coming from. This is Cape Grim in Tasmania, run by the
      Bureau of Meteorology, where CSIRO scientists collect what is
      probably the most pristine air in the world. From the air
      collected here, they can determine which pollutants there are in
      the atmosphere. Their measurements had indicated that damage to
      the ozone layer would decline over the next decade, but the
      discovery of the damaging Halon 1202 has made them reassess that
      time-table.
      
</desc>
</clip>

<clip id="research" start="npt:58">
<desc>
Halon 1202 has an ozone depletion potential approximately
      half that of the common CFC's, but it has increased six fold in
      the atmosphere since the late 1970's. In fact, during the past
      two years the atmospheric concentration of halon 1202 has been
      growing 17 per cent per year.
      
</desc>
</clip>

<clip id="scientist" start="npt:0:1:20">
<desc>
Dr Paul Fraser: "It's present in the atmosphere in quite
      small amounts currently, but the potential exists that it could
      rise quite rapidly and it is quite damaging to the ozone
      layer."
      
</desc>
</clip>

<clip id="ideas" start="npt:0:1:31">
<desc>
The halon 1202 could be a by-product from the production
      of other known halons, or it may indeed be being made
      deliberately for military applications.
      
</desc>
</clip>

<clip id="scientist_2" start="npt:0:1:43">
<desc>
Dr Paul Fraser: "The best way to stop halons increasing in
      the atmosphere is to have them included in the Montreal
      Protocol, which is an international agreement designed to limit
      the release of ozone depleting chemicals to the atmosphere."
      
</desc>
</clip>

<clip id="montreal_protocol" start="npt:0:1:57">
<desc>
When the Montreal Protocol was drawn up in 1987, only
      three halons were included, Halon 1202 was then unknown. And
      while the phasing out of CFC's and the known Halons is beginning
      to show results, the new Halon's growth could set back ozone
      recovery by up to ten years.
      
</desc>
</clip>

<clip id="end" start="npt:0:2:18">
<a href="mailto:pjf@dar.csiro.au">
Email Dr. Paul Fraser</a>
<desc>
For more information on Grim News on the Ozone Layer please contact:
      Dr Paul Fraser
      Atmospheric Research
      Private Bag 1
      Aspendale
      Victoria 3195
      
</desc>
</clip>

</cmml>
