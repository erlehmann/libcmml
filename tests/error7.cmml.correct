<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<!DOCTYPE cmml SYSTEM "cmml.dtd">

<cmml>

<head>
<title>
Australia Advances - Grim News on the Ozone Layer
</title>
<meta name="DESCRIPTION" content="The recovery of the ozone layer is being slowed dramatically by the presence and production of the gas, halon"/>
<meta name="KEYWORDS" content="Ozone, Halon"/>
<meta name="TYPE" content="video"/>
</head>

<clip id="intro" track="default" start="npt:0.000">
<a href="http://www.csiro.au/promos/ozadvances/">
http://www.csiro.au/promos/ozadvances/</a>
<desc>
Introduction to 'Grim News on the Ozone Layer - Australia Advances video'.
</desc>
</clip>

