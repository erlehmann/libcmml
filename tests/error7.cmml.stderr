error7.cmml:23:0: head or clip tag before stream tag not allowed
